todo_app/app.py

```diff
@@ -13,59 +17,167 @@ def create_app():
     mongo_db_client = pymongo.MongoClient(os.getenv('DATABASE_CONNECTION_STRING'))
     db = mongo_db_client[os.getenv('DATABASE_NAME')]
     collection = db['todo_app_items']
+    users_collection = db['users']
 
     app = Flask(__name__)
     app.config.from_object('todo_app.flask_config.Config')
+    app.secret_key = os.getenv('SECRET_KEY')
+    login_manager = LoginManager()
+
+    web_application_client = WebApplicationClient(os.getenv('CLIENT_ID'))
+
+    class User(UserMixin):
+        def __init__(self, id):
+            self.id = id
```

Comment:
Would be nicer to make name and role part of the User class rather than having to set those values afterwards each time you construct the User.